waiter/integration/waiter/token_request_test.clj

```diff
@@ -213,40 +252,92 @@
           (finally
             (delete-token-and-assert waiter-url token)))))))
 
+(deftest ^:parallel ^:integration-fast test-token-sync-unaffected-by-run-as-user-permissions
```

Comment:
Waiter still needs to add support for a sync call, which will be made by an external process. This test intends to verify that support works.